test/xspec.bats

```diff
@@ -1444,6 +1487,33 @@ load bats-helper
     [ "${lines[17]}" = "passed: 4 / pending: 0 / failed: 0 / total: 4" ]
 }
 
+@test "Saxon 11 and later: invoking xspec using SAXON_HOME finds Saxon jar and XML Catalog Resolver jar" {
+    if [ -z "${BUNDLED_XML_RESOLVER}" ]; then
+        skip "Saxon 11 and later only"
+    fi
+    export SAXON_HOME="${work_dir}/saxon ${RANDOM}"
+    mkdir "${SAXON_HOME}"
+    mkdir "${SAXON_HOME}/lib"
+    cp "${SAXON_JAR}" "${SAXON_HOME}"
+    saxon_jar_dir="$(dirname -- "${SAXON_JAR}")"
+    ls "${saxon_jar_dir}"/lib
+    cp "${saxon_jar_dir}"/lib/xmlresolver* "${SAXON_HOME}"/lib/
+    cp "${saxon_jar_dir}"/lib/jline* "${SAXON_HOME}"/lib/
```

Comment:
These two `cp` lines can be just

```suggestion
    cp "${saxon_jar_dir}"/lib/* "${SAXON_HOME}/lib/"
```

Reply:
Thanks for the suggestions. Fixed in 7f34f8b2b728ce250d5bf7c3d5ea19bd8d78de80.